controllers/generateConfigController.js

```diff
@@ -1,9 +1,9 @@
-const fs = require("fs");
-const { validEngines } = require("../utils/translationEnginesUtils");
-const { confirmUserAction, promptUserInput } = require("../utils/promptUtils");
-const { parsePath } = require("../utils/getConfigPath");
-const { listFilesOnDirectory } = require("../utils/listFiles");
-const { supportedLanguagesCodes } = require("../utils/supportedLanguagesUtils");
+import * as fs from "fs";
```

Comment:
Why not import fs from "fs"

Reply:
it gives me an error, here is the copilot explanation

![image](https://github.com/heliomar-pena/i18n-populator/assets/66505715/22b99017-cdce-496f-8a7b-7f8c57094e9a)

Reply:
Weird as is working in other places like controllers/translateController.js

Maybe something related to modules or something like that?

Reply:
hmm you're right. Maybe I got confused and in this case it was not needed too.

Will double-check. Thanks for the correction!

Reply:
Yea you was right! Was a tests that I was doing to FS which makes that error appear. Already fixed it and now it works as expected